### PR TITLE
Updating samtools

### PR DESCRIPTION
**Pull request by asp8200**
In this PR, we just update the "pure" samtools-modules. The modules using mulled containers with samtools will be updated in a later PR.

### CODE REVIEW

**Review by maxulysse — COMMENTED** [review]
Looking good already.
Don't forget to update CHANGELOG

**Comment by maxulysse** [comment]
ok, good news, we fixed the samtools/view tests

**Comment by asp8200** [comment]
Is it a similar problem with samtools/mpileup?

**Comment by maxulysse** [comment]
> Is it a similar problem with samtools/mpileup?

Unfortunately no

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,8 +23,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#1359](https://github.com/nf-core/sarek/pull/1359) - Removing params usage from local modules
 - [#1359](https://github.com/nf-core/sarek/pull/1359) - Removing params usage from local subworkflows
 - [#1360](https://github.com/nf-core/sarek/pull/1360) - Sync `TEMPLATE` with `tools` `2.11`
+- [#1408](https://github.com/nf-core/sarek/pull/1408) - Updating samtools to v1.19.2 in "pure" samtools-modules, that is, samtools in mulled containers have NOT yet been updated.
 - [#1411](https://github.com/nf-core/sarek/pull/1411) - Temporally disable sentieon related tests
 
+
```
Comment (maxulysse) [inline CHANGELOG.md:29]:
```suggestion
```

File: CHANGELOG.md
Comment (maxulysse) [inline CHANGELOG.md:1]: Can you add the samtools version update in the related section?

**Pull request merged**